### DISCUSSION TITLE
Support 'datetime' and 'datetime-range' filter variants

### DISCUSSION THREAD

**Discussion by noam3127** [KevinVandy/material-react-table#882] [Ideas]
I'm wondering if there's any interest in having 'datetime' and 'datetime-range' filter variants. My team has a use-case for this. These would be almost identical to the existing 'date' and 'date-range' variants, but using `@mui/x-date-pickers/DateTimePicker` instead of `@mui/x-date-pickers/DatePicker`.  

I can try making a PR if others deem this feature useful as well.

**Comment by KevinVandy** [comment]
A simple PR with as much code reuse as possible would be welcome

**Comment by noam3127** [comment]
https://github.com/KevinVandy/material-react-table/pull/905

**Comment by KevinVandy** [comment]
datetime, datetime-range, time, and time-range variants have all been added in v2.4
